### PR TITLE
[T3A3][T15-A2] Muhammad Irwan Bin Hussin

### PR DESCRIPTION
**Pull request by nawri14**

### CODE REVIEW

**Comment by nawri14** [comment]
Ready for Review

**Comment by K1ang** [comment]
@nawri14 Some comments added. Please close the PR after reading comments.
- Good job in encapsulating the variable.
- Keep it up.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -37,19 +54,22 @@ public static boolean isValidAddress(String test) {
...
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
+                && this.block.equals(((Address) other).block))
+                && this.street.equals(((Address) other).street)
+                && this.unit.equals(((Address) other).unit)
+                && this.postal.equals(((Address)other).postal); // state check
     }
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Address.java]: Good job in modifying the .equals method correspondingly to how the variables are handled. However, I believe when using .equals for those, you will have to override it in each class as well.

**Comment by K1ang** [comment]
@nawri14 Some comments added. Please close the PR after reading comments.
- Good job in expanding out the classes for the address.
- Great to see relevant documentations and test cases are updated.

**Pull request closed without merging**